### PR TITLE
fix: [PIE-4270]: serverless testcases fixed

### PR DESCRIPTION
**Pull request by satyam-harness**
##### Jira Links:
https://harness.atlassian.net/browse/PIE-4270

##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Review by Sagar-Rajput7 — DISMISSED** [review]

**Review by chetanvekariya-harness — COMMENTED** [review]
I think snapshot should have changed for this change. Earlier component was not rendering and now it is getting rendered then snapshot should change.

File: src/modules/75-cd/components/PipelineSteps/ServerlessAwsLambdaServiceSpec/__tests__/ServerlessAwsLambdaServiceSpec.test.tsx [hunk 1]
```diff
@@ -39,6 +39,24 @@ const fetchConnectors = (): Promise<unknown> => Promise.resolve({})
 jest.mock('services/cd-ng', () => ({
   useGetConnectorListV2: jest.fn().mockImplementation(() => ({ mutate: fetchConnectors })),
   getConnectorListV2Promise: () => Promise.resolve(mockManifestConnector),
+  useGetBuildDetailsForArtifactoryArtifactWithYaml: jest.fn().mockReturnValue({
```
Comment (chetanvekariya-harness) [inline src/modules/75-cd/components/PipelineSteps/ServerlessAwsLambdaServiceSpec/__tests__/ServerlessAwsLambdaServiceSpec.test.tsx:42]: This is response type of this call - `ArtifactoryResponseDTO`. Please check data type and mock response properly.

**Review by chetanvekariya-harness — CHANGES_REQUESTED** [review]

File: src/modules/75-cd/components/PipelineSteps/ServerlessAwsLambdaServiceSpec/__tests__/ServerlessAwsLambdaServiceSpec.test.tsx (hunk 1 above)
Comment (satyam-harness) [reply src/modules/75-cd/components/PipelineSteps/ServerlessAwsLambdaServiceSpec/__tests__/ServerlessAwsLambdaServiceSpec.test.tsx:42]: addressed

**Comment by satyam-harness** [comment]
retrigger build

**Comment by satyam-harness** [comment]
retrigger build

**Comment by satyam-harness** [comment]
retrigger build

File: src/modules/75-cd/components/PipelineSteps/ServerlessAwsLambdaServiceSpec/__tests__/__snapshots__/ServerlessAwsLambdaServiceSpec.test.tsx.snap [hunk 2]
```diff
@@ -128,6 +128,510 @@ exports[`ServerlessAwsLambdaServiceSpec tests When stepViewType is InputSet rend
...
+      </div>
+      <div
+        class="nopadLeft accordionSummary"
+        id="Stage.undefined.Service.Artifacts.Sidecars"
+      />
+    </div>
+    <p
+      class="StyledProps--font StyledProps--main"
+    >
+      Errors
```
Comment (chetanvekariya-harness) [inline src/modules/75-cd/components/PipelineSteps/ServerlessAwsLambdaServiceSpec/__tests__/__snapshots__/ServerlessAwsLambdaServiceSpec.test.tsx.snap:397]: Could you please check why this is appearing ?

File: src/modules/75-cd/components/PipelineSteps/ServerlessAwsLambdaServiceSpec/__tests__/__snapshots__/ServerlessAwsLambdaServiceSpec.test.tsx.snap [hunk 3]
```diff
@@ -128,6 +128,510 @@ exports[`ServerlessAwsLambdaServiceSpec tests When stepViewType is InputSet rend
...
+                >
+                  <div
+                    class="bp3-input-group MultiTypeInput--input"
+                  >
+                    <input
+                      autocomplete="off"
+                      class="bp3-input"
+                      name=".artifacts.primary.spec.repository"
+                      type="text"
+                      value=""
```
Comment (chetanvekariya-harness) [inline src/modules/75-cd/components/PipelineSteps/ServerlessAwsLambdaServiceSpec/__tests__/__snapshots__/ServerlessAwsLambdaServiceSpec.test.tsx.snap:571]: After this, field for artifactPathFilter should appear as well.

**Review by chetanvekariya-harness — CHANGES_REQUESTED** [review]

**Pull request closed without merging**